cardano-cli/src/Cardano/CLI/EraBased/Options/Governance/Query.hs

```diff
@@ -65,7 +65,7 @@ pGovernanceQueryDRepStateCmd env era = do
   pure
     $ subParser "drep-state"
     $ Opt.info (GovernanceQueryDRepStateCmd cOn <$> pDRepStateQueryCmd)
-    $ Opt.progDesc "Get the DRep state"
+    $ Opt.progDesc "Get the DRep state. If no DRep credentials are provided, return stake distributions for all of them."
```

Comment:
I'm not sure what the accepted terminology is here. Should it be "voting stake distributions"?

Reply:
I've used ledger terminology here, which seems to be fine according to  https://github.com/input-output-hk/cardano-cli/pull/281#discussion_r1326052618